src/frontend/components/SøknadsSteg/OmDeg/useOmdeg.tsx

```diff
@@ -58,33 +56,16 @@ export const useOmdeg = (): {
         borPåRegistrertAdresse.verdi === ESvar.NEI
     );
 
-    const komTilNorgeDato = useDatovelgerFeltMedJaNeiAvhengighet(
-        søker.komTilNorgeDato,
-        ESvar.NEI,
-        værtINorgeITolvMåneder,
-        'omdeg.opphold-sammenhengende.dato.feilmelding',
-        dagensDato()
-    );
-
-    const planleggerÅBoINorgeTolvMnd = useFelt<ESvar | null>({
-        feltId: søker.planleggerÅBoINorgeTolvMnd.id,
-        verdi: søker.planleggerÅBoINorgeTolvMnd.svar,
-        valideringsfunksjon: (felt: FeltState<ESvar | null>) => {
-            return felt.verdi
-                ? ok(felt)
-                : feil(
-                      felt,
-                      <SpråkTekst id={'omdeg.planlagt-opphold-sammenhengende.feilmelding'} />
-                  );
-        },
-        skalFeltetVises: avhengigheter => {
-            return (
-                avhengigheter.værtINorgeITolvMåneder &&
-                avhengigheter.værtINorgeITolvMåneder.verdi === ESvar.NEI
-            );
+    const planleggerÅBoINorgeTolvMnd = useJaNeiSpmFelt(
+        søker.planleggerÅBoINorgeTolvMnd,
+        'omdeg.planlagt-opphold-sammenhengende.feilmelding',
+        {
+            ...(!søker.adressebeskyttelse && {
```

Comment:
Må vi fjerne spørsmålet for de med adressebeskyttelse?

Reply:
Bare kopierte logikken fra spm om de har vært i norge i 12 måneder eller ikke, skal spørre Daniel